patches/archiver+6.0.2.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/node_modules/archiver/lib/core.js b/node_modules/archiver/lib/core.js
+index 7c0a74d..5ac4e3a 100644
+--- a/node_modules/archiver/lib/core.js
++++ b/node_modules/archiver/lib/core.js
+@@ -631,7 +631,8 @@ Archiver.prototype.directory = function(dirpath, destpath, data) {
+ 
+   var globOptions = {
+     stat: true,
+-    dot: true
++    dot: true,
++    follow: true,
```

Comment:
We could consider using the same code from your node-archiver PR: https://github.com/archiverjs/node-archiver/pull/810

Reply:
Done as part of [f10e4c8](https://github.com/Automattic/studio/pull/1447/commits/f10e4c852065b2d7e6147e5abb167ee8689f8000)